watershed_extent_update.Rmd

```diff
@@ -16,111 +16,130 @@ The purpose of this document is to establish an agreed upon spatial extent for e
 
 Data source table: 
 
-|Database: | Use:     |
-| ---------|----------|
-|[National Hydrological Dataset - high res](https://www.usgs.gov/national-hydrography/nhdplus-high-resolution) - water body attributes | X 
+| Feature  |Source/Database | Notes     |
+| ---------|----------|-----------| 
+|Water Bodies |[National Hydrological Dataset - high res](https://www.usgs.gov/national-hydrography/nhdplus-high-resolution) - water body attributes | Carson sink was manually drawn 
```

Comment:
This is good. also make sure to specifically state where each of the lake shapes came from, there are a couple out of medium res, correct?